runtime/lua/vim/lsp.lua

```diff
@@ -17,6 +17,7 @@ local if_nil = vim.F.if_nil
 
 local lsp = {
   protocol = protocol,
+  _inlay_hint = require('vim.lsp.inlay_hint'),
```

Comment:
if it's private, should be prefixed with underscore, right?
```suggestion
  _inlay_hint = require('vim.lsp._inlayhint'),
```

Reply:
done